sql/parser/all_keywords.awk

```diff
@@ -6,7 +6,7 @@ BEGIN {
     print "var keywords = map[string]int{"
 
     # This variable will be associated with a pipe for intermediate output.
-    sort = "sort"
+    sort = "env LC_ALL=C sort | xargs -i printf '\"%s\": %s,\n' {} {}"
```

Comment:
`xargs -i` is not portable (OSX `xargs` doesn't have it). 

If it were important that the output be in order, then I would agree that we should sort the keywords without their surrounding quotes. However, we don't really care that it's in any particular order, we just want that order to be stable, so it's better to do the simplest thing and sort the whole line.